packages/web/components/cards/bond-card.tsx

```diff
@@ -189,135 +193,164 @@ const Drawer: FunctionComponent<{
   drawerUp: boolean;
   toggleDetailsVisible: () => void;
   onGoSuperfluid: () => void;
-}> = ({
-  duration,
-  aggregateApr,
-  swapFeeApr,
-  swapFeeDailyReward,
-  incentivesBreakdown,
-  superfluid,
-  drawerUp,
-  toggleDetailsVisible,
-}) => {
-  const uniqueCoinImages = useMemo(() => {
-    const imgSrcDenomMap = new Map<string, string>();
-    incentivesBreakdown.forEach((breakdown) => {
-      const currency = breakdown.dailyPoolReward.currency;
-      if (currency.coinImageUrl) {
-        imgSrcDenomMap.set(currency.coinDenom, currency.coinImageUrl);
-      }
-    });
-    return Array.from(imgSrcDenomMap.values());
-  }, [incentivesBreakdown]);
-  const t = useTranslation();
-
-  return (
-    <div
-      className={classNames(
-        "absolute -bottom-[234px] left-1/2 z-40 flex h-[320px] w-full -translate-x-1/2 flex-col transition-all duration-300 ease-inOutBack",
-        {
-          "-translate-y-[220px] rounded-t-[18px] bg-osmoverse-700": drawerUp,
+}> = observer(
+  ({
+    duration,
+    aggregateApr,
+    swapFeeApr,
+    swapFeeDailyReward,
+    incentivesBreakdown,
+    superfluid,
+    drawerUp,
+    toggleDetailsVisible,
+  }) => {
+    const {
+      queriesStore,
+      chainStore: {
+        osmosis: { chainId },
+      },
+    } = useStore();
+    const uniqueCoinImages = useMemo(() => {
+      const imgSrcDenomMap = new Map<string, string>();
+      incentivesBreakdown.forEach((breakdown) => {
+        const currency = breakdown.dailyPoolReward.currency;
+        if (currency.coinImageUrl) {
+          imgSrcDenomMap.set(currency.coinDenom, currency.coinImageUrl);
         }
-      )}
-    >
+      });
+      return Array.from(imgSrcDenomMap.values());
+    }, [incentivesBreakdown]);
+    const t = useTranslation();
+
+    const queriesCosmos = queriesStore.get(chainId).cosmos;
+    const inflation = queriesCosmos.queryInflation;
```

Comment:
Don't we want to get the inflation of the counterparty token? LMK if I'm misunderstanding.. 

I think we can get the chain ID from the coin denom with this function;
https://github.com/osmosis-labs/osmosis-frontend/blob/efc03405f304615602ca199917c7ec2bcbde1b39/packages/stores/src/chain.ts#L60

Reply:
Calculating inflation accurately for each token chain can be challenging, as some tokens, such as `WBTC`, `USDC`, and `FOX`, are not directly associated with the chain's inflation. Johnny's method uses the Osmosis chain's inflation as a reference point. Here's a graph he made showcasing that high inflation pools would be identified with this baseline. The red line represents Osmosis Staking APR times 5. 
![image](https://user-images.githubusercontent.com/21092519/231352947-a44a3349-6dae-45ae-b2b1-12978125610f.png)